front/app/modules/commercial/insights/admin/containers/Insights/Edit/index.tsx

```diff
@@ -231,6 +239,13 @@ const EditInsightsView = ({
     (category) => category.id === query.category
   );
 
+  const onSearch = (search: string) => {
+    clHistory.push({
```

Comment:
Shall we maybe replace here ? That's going to be a very long history

Reply:
We might want to debounce the search, with bigger amount of data this might actually do all the queries and get slow.

Reply:
The search input component already has a debounce of 500 by default. Do you think we need more than that?

Reply:
No, that sounds good indeed, didn't know, sorry :)